package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage",
```

Comment:
Is this good practice? With npm 3 (flat structure), could we install minimatch as our own dependency, thus overriding the version with a non-vulnerable version?